### ISSUE TITLE
ItemSpray effect seems to be broken

### ISSUE THREAD

**Issue by Pendragon1111** [TheComputerGeek2/MagicSpells#96]
While attempting to use it, noticed it doesn't seem to work at all. 
```
      4:
       position: special
       effect: itemspray
       type: 397
       data: 1
       quantity: 10
       duration: 20
       force: 2.0
```
Was my code, and even just trying "type: 1" or "type: stone" didn't work. My guess is that its not available somehow in the latest version. 

Using:
Spigot: 1.12.2
Magicspells: 3.5-dev-1802072024
Effectlib: 5.5

**Comment by RifleDLuffy** [comment]
I've encountered the same sort of problems. I'll look into it aswell

**Comment by Chronoken** [comment]
Fixed in 4.0
